suite-native/transactions/src/components/TransactionDetail/NetworkTransactionDetailSummary.tsx

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import { useSelector } from 'react-redux';
+
+import { A } from '@mobily/ts-belt';
+
+import {
+    TransactionsRootState,
+    selectTransactionByTxidAndAccountKey,
+} from '@suite-common/wallet-core';
+import { AccountKey } from '@suite-common/wallet-types';
+import { ErrorMessage, VStack } from '@suite-native/atoms';
+
+import { selectTransactionInputAddresses, selectTransactionOutputAddresses } from '../../selectors';
+import { TransactionDetailAddressesSection } from './TransactionDetailAddressesSection';
+import { TransactionDetailStatusSection } from './TransactionDetailStatusSection';
+
+export const NetworkTransactionDetailSummary = ({
+    accountKey,
+    txid,
+    onShowMore,
+}: {
+    accountKey: AccountKey;
+    txid: string;
+    onShowMore: () => void;
+}) => {
+    const transaction = useSelector((state: TransactionsRootState) =>
+        selectTransactionByTxidAndAccountKey(state, txid, accountKey),
+    );
+    const transactionInputAddresses = useSelector((state: TransactionsRootState) =>
+        selectTransactionInputAddresses(state, txid, accountKey),
+    );
+    const transactionOutputAddresses = useSelector((state: TransactionsRootState) =>
+        selectTransactionOutputAddresses(state, txid, accountKey),
+    );
+
+    if (
+        !transaction ||
+        A.isEmpty(transactionInputAddresses) ||
+        A.isEmpty(transactionOutputAddresses)
+    )
```

Comment:
please add `{ }` to this if, these two returns right above each other looks really strange

Reply:
fixed in 1e27599ac